Engine/AssemblyFinder.cs

```diff
@@ -0,0 +1,139 @@
+﻿//            Copyright Keysight Technologies 2012-2019
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Diagnostics;
+using Tap.Shared;
+
+namespace OpenTap
+{
+    class AssemblyFinder
+    {
+        static TraceSource log = Log.CreateSource("AssemblyFinder");
+        public void Invalidate()
+        {
+            lastSearch = DateTime.MinValue;
+        }
+
+        public bool IncludeDependencies = false;
+        public bool OutputLog = true;
```

Comment:
could this just be 'Quiet'? I think boolean properties should be false by default (when possible).